tests/fixtures/response/config/environment.js

```diff
@@ -0,0 +1,51 @@
+/* jshint node: true */
```

Comment:
Only need to set this if you were getting `host` off the request, you can remove this